app/controllers/api/v0/user_collections_controller.rb

```diff
@@ -0,0 +1,13 @@
+module Api
+  module V0
+    class UserCollectionsController < ApiController
+      def index
+        collections = UserCollection.where(user_id: session_current_user_id).pluck(:title)
+        if !collections.include?("Default Collection")
+          collections.prepend("Default Collection")
```

Comment:
This was a great idea to create a default collection for user's to save articles to. Is there a better name for this than "Default Collection"?